src/schema/v2/notifications/index.ts

```diff
@@ -98,6 +99,68 @@ export const NotificationType = new GraphQLObjectType<any, ResolverContext>({
         return formatDate(date, dateFormat, timezone)
       },
     },
+    previewImages: {
+      type: new GraphQLList(Image.type),
```

Comment:
Should there be any non-nullability constraints added?

Reply:
yeah, this could be changed to because the resolver wraps the result with `compact`.

```suggestion
      type: new GraphQLNonNull(GraphQLList(GraphQLNonNull(Image.type))),

```